python-package/securexgboost/core.py

```diff
@@ -1175,6 +1175,13 @@ def add_client_key(self, data, data_len, signature, sig_len):
         # Add client key
         _LIB.add_client_key(data, data_len, signature, sig_len)
 
+
+    def sync_client_key(self):
```

Comment:
See comment above for `c_api.h`. Not sure if we need to expose this function to the client.